index.html

```diff
@@ -528,43 +416,25 @@ <h1>DID Resolution</h1>
 		<dd>
 			A <a href="#metadata-structure">metadata structure</a> consisting of values
 			relating to the results of the <a>DID resolution</a> process which typically
-			changes between invocations of the <code>resolve</code> and
-			<code>resolveRepresentation</code> functions, as it represents data about the
+			changes between invocations of the <code>resolve</code>
+			function, as it represents data about the
 			resolution process itself. This structure is REQUIRED, and in the case of an
 			error in the resolution process, this MUST NOT be empty. This metadata is
-			defined by <a href="#did-resolution-metadata"></a>. If
-			<code>resolveRepresentation</code> was called, this structure MUST contain a
-			<code>contentType</code> property containing the Media Type of the
-			representation found in the <code>didDocumentStream</code>. If the resolution is
+			defined by <a href="#did-resolution-metadata"></a>. If the resolution is
 			not successful, this structure MUST contain an <code>error</code> property
 			describing the error.
 		</dd>
 		<dt>
 			<dfn>didDocument</dfn>
 		</dt>
 		<dd>
-			If the resolution is successful, and if the <code>resolve</code> function was
-			called, this MUST be a <a>DID document</a> abstract data model (a <a
-				data-cite="INFRA#maps">map</a>) as described in <a href="https://w3c.github.io/did-core/#data-model"></a> that
-			is capable of being transformed into a <a href="https://w3c.github.io/did-core/#dfn-did-documents">conforming DID Document</a>
-			(representation), using the production rules specified by the representation.
+			If the resolution is successful, this MUST be a <a>DID document</a> that
+			is capable of being represented in one of the conformant
+			<a href="https://w3c.github.io/did-core/#representations">representations</a>.
```

Comment:
```suggestion
			<a href="https://w3c.github.io/did/#representations">representations</a>.
```

The URL https://w3c.github.io/did-core gives a 404 (for me at least). Maybe that means the redirection is not set up properly idk. But /did works fine.